app/views/documents/index/_results.html.erb

```diff
@@ -28,13 +29,12 @@
               },
               metadata: {
                 document_type: document.document_type_schema.label,
-                document_create: t("documents.index.search_results.creator", creator: document.creator&.name || I18n.t!("documents.index.search_results.unknown_user")),
```

Comment:
We've lost the `document_create` on the way or we dropped it intentionally?

Reply:
Hey alex - we dropped it intentionally. @soniaturcotte said we don't need both creator and last editor here.

Reply:
OK